src/listener/hastebinConversion.ts

```diff
@@ -0,0 +1,51 @@
+import { Listener } from 'discord-akairo';
+import { Message } from 'discord.js';
+import fetch from 'node-fetch';
+import { guildConfigs } from '../guild/config/guildConfigs';
+import hastebin from 'hastebin-gen';
+import { MessageEmbed } from 'discord.js';
+
+export default class HastebinConversionListener extends Listener {
+	constructor() {
+		super('hastebinConversion', {
+			emitter: 'client',
+			event: 'message',
+		});
+	}
+
+	async exec(msg: Message) {
+		if (!msg.guild) return;
+
+		const hastebinConversionConfig = guildConfigs.get(msg.guild.id)?.features
+			.hastebinConversion;
+		if (
+			!hastebinConversionConfig ||
+			!hastebinConversionConfig.channels.includes(msg.channel.id)
+		)
+			return;
+
+		const messageAttachment = msg.attachments.find(attachment =>
+			hastebinConversionConfig.whitelistedExtensions.some(ext =>
+				attachment.name?.endsWith(ext)
```

Comment:
```suggestion
				attachment.name?.endsWith(`.${ext}`)
```

Making sure this is actually an extension